pyart/util/columnsect.py

```diff
@@ -0,0 +1,294 @@
+"""
+Function for extracting the radar column above a target
+given position in lat, lon
+
+"""
+
+import numpy as np
+
+from ..core.transforms import antenna_vectors_to_cartesian
+
+
+def sphere_distance(rad_lat, tar_lat, rad_lon, tar_lon):
```

Comment:
Can you spell out `radar_latitude`, `target_latitude`... I think this would make it more clear to the end user in terms of the API.